tests/react/dependency.test.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react/no-unescaped-entities */
```

Comment:
@dai-shi 

1. Maintain the current state.
2. Add `react/no-unescaped-entities: 'off'` in the rules.

This file only has a `react/no-unecaped-entities` error, so i added `eslint-disable `. Which option do you prefer?

Reply:
Yes, this looks fine for now.